DOCS-3003: Add prerequisite of physically assembling hardware to components and services (#3540)
viamrobotics/docs: docs/components/motor/gpiostepper.md

{{< /alert >}}

> To configure a `gpiostepper` motor as a component of your machine, first configure the [board](/components/board/) to which the motor driver is wired.
> Make sure to physically connect your motor to your board and turn it on.
> Then, add the motor:

Reviewer: ```suggestion
To configure a `gpiostepper` motor as a component of your machine, first make sure the motor is wired to a suitable stepper motor driver, which is in turn wired to a board.
Configure the [board](/components/board/) to which the motor driver is wired.
Then configure the motor:
```
